xscen/aggregate.py

```diff
@@ -29,6 +31,7 @@
 
 __all__ = [
     "climatological_mean",
```

Comment:
Is the goal to eventually get rid of `climatological_mean`  to only keep `op` ? If so, there should be a future warning. 

If we want to keep it forever, I would modify the args to be the same as `op`. (But also keep the old version for a bit with a future warning).

Reply:
My suggestion is indeed to remove climatological_mean eventually. Agreed, there should be a future / deprecation warning.

Reply:
FutureWarning added.